### PR TITLE
!str #16563 terminal operations start with run*; runFold runForeach

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
resolves #16563

File: akka-stream-tests/src/test/scala/akka/stream/io/StreamTcpSpec.scala [hunk 1]
```diff
@@ -3,16 +3,15 @@
...
-import akka.util.ByteString
-import akka.stream.scaladsl.Flow
+import akka.stream.scaladsl.{ Flow, _ }
 import akka.stream.testkit.AkkaSpec
-import akka.stream.scaladsl._
 import akka.stream.testkit.TestUtils.temporaryServerAddress
+import akka.util.ByteString
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
```
Comment (bantonsson) [inline akka-stream-tests/src/test/scala/akka/stream/io/StreamTcpSpec.scala]: IDEA is not your friend :wink:

**Comment by bantonsson** [comment]
LUTM, but I guess that this shed has already sailed :wink:

**Comment by patriknw** [comment]
LGTM

**Comment by drewhk** [comment]
LNTM ;)

File: akka-stream-tests/src/test/scala/akka/stream/io/StreamTcpSpec.scala (hunk 1 above)
Comment (ktoso) [inline akka-stream-tests/src/test/scala/akka/stream/io/StreamTcpSpec.scala]: Fixed (file and more intellij tweaking)

**Comment by ktoso** [comment]
LWatTM? ;)

**Comment by drewhk** [comment]
(Neutral)

**Pull request merged**